cctrl/auth.py

```diff
@@ -92,12 +130,71 @@ def delete_tokenfile(settings):
     return False
 
 
+def set_configfile(settings, email=None, ssh_auth=None, ssh_path=None):
+    create_config_dir(settings)
+    content = get_configfile(settings)
+
+    if email:
+        content['email'] = email
+
+    if ssh_auth is not None:
+        content['ssh_auth'] = ssh_auth
+
+    if ssh_path:
+        content['ssh_path'] = ssh_path
+
+    userfile = open(settings.config_path, 'w')
+    json.dump(content, userfile)
```

Comment:
I would prefer if we store the config in yaml or config format. It is much easier to write and read by hand if you want to.